zyte_common_items/components.py

```diff
@@ -148,3 +162,122 @@ class Metadata(Item):
     #: `0`. When there is no complete certainty, the value could be anything in
     #: between (e.g. `0.96`).
     probability: Optional[float] = None
+
+
+@attrs.define(kw_only=True)
+class BusinessPlaceMetadata(Metadata):
+    """Data extraction process metadata.
+
+    See :class:`BusinessPlace.metadata <zyte_common_items.items.BusinessPlace.metadata>`.
+    """
+
+    #: The search text the place was found with.
+    searchText: Optional[str] = None
+
+
+@attrs.define(kw_only=True)
+class Address(Item):
+    """Address item."""
+
+    #: The raw address information, as it appears on the website.
+    addressRaw: Optional[str] = None
+
+    #: The street address of the place.
+    streetAddress: Optional[str] = None
+
+    #: The city the place is located in.
+    addressCity: Optional[str] = None
+
+    #: The locality to which the place belongs.
+    addressLocality: Optional[str] = None
+
+    #: The region of the place.
+    addressRegion: Optional[str] = None
+
+    #: The country the place is located in.
+    #:
+    #: The country name or the two-letter ISO 3166-1 alpha-2 country code.
+    #: https://en.wikipedia.org/wiki/ISO_3166-1
```

Comment:
```suggestion
    #: The country name or the `ISO 3166-1 alpha-2 country code 
    #: <https://en.wikipedia.org/wiki/ISO_3166-1>`__.
```

“two-letter” and “alpha-2” are the same thing.

This reminds me, you should also update the docs with references to the new classes. You can build the docs locally with `tox -e docs`, and open `.tox/docs/tmp/html/index.html` in a web browser to see the result, which for reference documentation will include these `#:` comments.